composer.json

```diff
@@ -13,7 +13,8 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^5.6 || ^7.0",
+        "scriptfusion/static-class": "^1"
```

Comment:
* I understand this is a core package, however this is also a common pattern and it would be my intention to introduce this trait to the wider framework since the static utility class pattern is common enough to warrant this.
* I believe a private constructor is at least 3LOC if you respect PSR-2; 4LOC if you annotate that the empty constructor is intentional as in the trait.
* I do not understand what you mean. `StaticClass` is tagged as 1.0.0 (stable). If this is not good enough what else do you need?